gamestonk_terminal/portfolio_optimization/README.md

```diff
@@ -1,169 +1,192 @@
+* --pie : Display a pie chart for weights
 
-The usage is:
-````
-max_sharpe [-p PERIOD] [-v --value VALUE] [--pie] 
-````
-* -p/--period Amount of time to retrieve data from yfinance. Options are: 1d,5d,1mo,3mo,6mo,1y,2y,5y,10y,ytd,max and it defaults to 3mo.
-* -v/--value If provided, this represents an actual allocation amount for the portfolio.  Defaults to 1, which just returns the weights.
-* --pie Flag that displays a pie chart of the allocations.
+![equal](https://user-images.githubusercontent.com/25267873/114958292-9771e080-9e5a-11eb-9de6-1c2dfbe1e27b.png)
 
-### min_vol
-This portfolio minimizes the total volatility, which also means it has the smallest returns among the efficient frontier.
-The usage is:
-````
-min_vol [-p PERIOD] [-v --value VALUE] [--pie]
-````
-* -p/--period Amount of time to retrieve data from yfinance. Options are: 1d,5d,1mo,3mo,6mo,1y,2y,5y,10y,ytd,max and it defaults to 3mo.
-* -v/--value If provided, this represents an actual allocation amount for the portfolio.  Defaults to 1, which just returns the weights.
-* --pie Flag that displays a pie chart of the allocations.
 
-### eff_risk
-This portfolio maximizes the returns at a given risk tolerance
-The usage is:
+### property <a name="property"></a>
+
+```text
+property [-p PROPERTY] [-v VALUE] [--pie]
+```
+
+Returns a portfolio that is weighted based on a selected property info
+
+* -p : Property info to weigh. Use one of: previousClose, regularMarketOpen, twoHundredDayAverage, trailingAnnualDividendYield, payoutRatio, volume24Hr, regularMarketDayHigh, navPrice, averageDailyVolume10Day, totalAssets, regularMarketPreviousClose, fiftyDayAverage, trailingAnnualDividendRate, open, toCurrency, averageVolume10days, expireDate, yield, algorithm, dividendRate, exDividendDate, beta, circulatingSupply, regularMarketDayLow, priceHint, currency, trailingPE, regularMarketVolume, lastMarket, maxSupply, openInterest, marketCap, volumeAllCurrencies, strikePrice, averageVolume, priceToSalesTrailing12Months, dayLow, ask, ytdReturn, askSize, volume, fiftyTwoWeekHigh, forwardPE, fromCurrency, fiveYearAvgDividendYield, fiftyTwoWeekLow, bid, dividendYield, bidSize, dayHigh, annualHoldingsTurnover, enterpriseToRevenue, beta3Year, profitMargins, enterpriseToEbitda, 52WeekChange, morningStarRiskRating, forwardEps, revenueQuarterlyGrowth, sharesOutstanding, fundInceptionDate, annualReportExpenseRatio, bookValue, sharesShort, sharesPercentSharesOut, fundFamily, lastFiscalYearEnd, heldPercentInstitutions, netIncomeToCommon, trailingEps, lastDividendValue, SandP52WeekChange, priceToBook, heldPercentInsiders, shortRatio, sharesShortPreviousMonthDate, floatShares, enterpriseValue, threeYearAverageReturn, lastSplitFactor, legalType, lastDividendDate, morningStarOverallRating, earningsQuarterlyGrowth, pegRatio, lastCapGain, shortPercentOfFloat, sharesShortPriorMonth, impliedSharesOutstanding, fiveYearAverageReturn, and regularMarketPrice.
```

Comment:
Damn you went all out here. A lot of these are kinda useless and better used as factors in the optimization.

Reply:
I thought the same, only removed the non numeric ones. We can have the marketCap, and divYield as commands if you think those are the only that kind of make sense

Or do like we do regression on pred menu. So,

mktcap
dividend
property

Reply:
The only reason I did those are since those are common portfolio ones.  As I mentioned, we can have these as factors for the next step of PO (we can take this discussion offline)

Reply:
Let me refactor for this one :)